package.json

```diff
@@ -375,7 +375,7 @@
     "@softarc/sheriff-core": "^0.18.0",
     "@types/chroma-js": "^2.4.4",
     "@types/puppeteer": "^7.0.0",
-    "@types/vscode": "~1.55.0",
+    "@types/vscode": "~1.95.0",
```

Comment:
As we're still supporting all VS Code version since 1.55, we can't update the types here.

https://github.com/material-extensions/vscode-material-icon-theme/blob/0628045c51901f4aa57853cfe1f97cfb40c13adb/package.json#L43

Reply:
Should be ignored from now on: https://github.com/material-extensions/vscode-material-icon-theme/commit/df0b75c7e4992c9d22040cc229c817ada4ba95d1